docs/guides/observability/tracing.adoc

```diff
@@ -162,8 +162,41 @@ For more information, see the https://www.w3.org/TR/trace-context/#security-cons
 == Tracing in Kubernetes environment
 When the tracing is enabled when using the {project_name} Operator, certain information about the deployment is propagated to the underlying containers.
 
-NOTE: There is no support for tracing configuration in {project_name} CR yet, so the `additionalOptions` can be used to the `tracing-enabled` property and other tracing options.
+=== Configuration via Keycloak CR
 
+You can change tracing configuration via Keycloak CR fields as follows:
+
+[source,yaml]
+----
+apiVersion: k8s.keycloak.org/v2alpha1
+kind: Keycloak
+metadata:
+  name: example-kc
+spec:
+  features:
+    enabled:
+      - opentelemetry
+  tracing:
+    enabled: true                             # default 'false'
+    endpoint: http://my-tracing:4317          # default 'http://localhost:4317'
+    serviceName: my-best-keycloak             # default 'keycloak'
+    protocol: http/protobuf                   # default 'grpc'
+    samplerType: parentbased_traceidratio     # default 'traceidratio'
+    samplerRatio: 0.01                        # default '1'
+    compression: gzip                         # default 'none'
+    resourceAttributes:
+      service.namespace: keycloak-namespace
```

Comment:
Nitpick: As we already include the namespace in `k8s.namespace.name` as described below, using `service.namespace` here might guide people to a situation where they have both in their traces. 

About `serviceName` - isn't the default the name of the CR, in this case `example-kc`?

I would shorten this to those parameters you would usually set, therefore skipping `protocol` and `compression`, and possibly `serviceName` as well.

Reply:
> Nitpick: As we already include the namespace in k8s.namespace.name as described below, using service.namespace here might guide people to a situation where they have both in their traces.

> About serviceName - isn't the default the name of the CR, in this case example-kc?

> I would shorten this to those parameters you would usually set, therefore skipping protocol and compression, and possibly serviceName as well.

Agree, thanks!